docs/source/getting_started/index.md

```diff
@@ -1,87 +1,92 @@
 # Quick Start
 
+In this guide, we'll walk through how you can use the Llama Stack (server and client SDK) to test a simple agent.
+A Llama Stack agent is a simple integrated system that can perform tasks by combining a Llama model for reasoning with
+tools (e.g., RAG, web search, code execution, etc.) for taking actions.
+In Llama Stack, we provide a server exposing multiple APIs. These APIs are backed by implementations from different providers.
 
 Llama Stack is a stateful service with REST APIs to support seamless transition of AI applications across different environments. The server can be run in a variety of ways, including as a standalone binary, Docker container, or hosted service. You can build and test using a local server first and deploy to a hosted endpoint for production.
 
-In this guide, we'll walk through how to build a RAG agent locally using Llama Stack with [Ollama](https://ollama.com/) to run inference on a Llama Model.
+In this guide, we'll walk through how to build a RAG agent locally using Llama Stack with [Ollama](https://ollama.com/)
+as the inference [provider](../providers/index.md#inference) for a Llama Model.
 
+## Step 1: Installation and Setup
 
-### 1. Start Ollama
 
+### i. Install and Start Ollama for Inference
+
+Install Ollama by following the instructions on the [Ollama website](https://ollama.com/download).
+
+To start Ollama run:
 ```bash
 ollama run llama3.2:3b --keepalive 60m
 ```
 
 By default, Ollama keeps the model loaded in memory for 5 minutes which can be too short. We set the `--keepalive` flag to 60 minutes to ensure the model remains loaded for sometime.
 
-```{admonition} Note
-:class: tip
-
-If you do not have ollama, you can install it from [here](https://ollama.com/download).
-```
-
-### 2. Run Llama Stack locally
-
-We use `uv` to setup a virtual environment and install the Llama Stack package.
+### ii. Install `uv` to Manage your Python packages
```

Comment:
how much prominence should we give to setting up python. Should we just point to the appropriate external docs instead of showing them here. Also, would it be good to use tabs to show different envs like venv/conda/docker?

Reply:
It's just a tradeoff. By linking it, we're implicitly saying we'll maintain the docs in favor of an easier onboarding for developers and I think that's the right move given the current stage of the project.

I'll add the conda/docker configurations as well.